nama_kudasai.py

```diff
@@ -208,7 +209,7 @@ def check_video(config, video_id, cached_liveness):
     # Time to run!
     log.info(f'Starting downloader for {video_id} ({video_details["title"]}))')
     subprocess.Popen(
-        [sys.executable, 'download.py', '--', video_id],
+        [sys.executable, os.path.join(os.path.dirname(__file__), 'download.py'), '--', video_id],
```

Comment:
Huh, was this needed?

Reply:
my hack for not running out of disk space in wherever dogger backs its container filesystems is to download/log to a directory which gets bind mounted at runtime and because I'm too lazy to edit path handling for the whole download process I use this data directory as the working directory. 

this means that the python scripts are no longer in the current directory, so rather than install.py them into the venv I decided to just refer to them by absolute path instead. I considered making this script executable with a shebang as well but I wanted to keep non-containerized Windows compatibility in the case where  you don't have python files associated with py launcher on doubleclick.